fbpic/fields/interpolation_grid.py

```diff
@@ -218,3 +229,24 @@ def divide_by_volume( self, fieldtype ) :
                 self.Jz *= self.invvol[np.newaxis,:]
             else:
                 raise ValueError('Invalid string for fieldtype: %s'%fieldtype)
+    
+    
+    
+class EnvelopeInterpolationGrid(InterpolationGrid):
+    
+    def __init__(self, Nz, Nr, m, zmin, zmax, rmax, use_cuda=False ) :
+        
+        InterpolationGrid.__init__(self, Nz, Nr, m, zmin, zmax, rmax, use_cuda=False)
+        
+        # Allocate the fields arrays
+        self.A = np.zeros( (Nz, Nr), dtype='complex' )
+        self.dtA = np.zeros( (Nz, Nr), dtype='complex' )
+    
```

Comment:
For now, the `erase` method is not needed, and so I would recommend to just remove it for the moment.